e2e/test/scenarios/onboarding/onboarding-checklist.cy.spec.ts

```diff
@@ -27,6 +26,44 @@ describe("Onboarding checklist page", () => {
       ).should("not.exist");
     });
   });
+
+  it("the link should be dismissible from the sidebar", () => {
+    cy.get("[data-accordion=true]")
+      .as("accordionElement")
+      .findByRole("heading", { name: "Start visualizing your data" })
+      .should("be.visible");
+
+    cy.findByTestId("main-navbar-root")
+      .findByRole("listitem", { name: "How to use Metabase" })
+      .as("sidebarOnboardingLink")
+      .realHover()
+      .icon("eye_crossed_out")
+      .as("dismissIcon")
+      .should("be.visible")
+      .trigger("mouseenter");
+    cy.findByRole("tooltip", { name: "Hide page" }).should("be.visible");
+    cy.get("@dismissIcon").click();
+
+    cy.log("The link should be removed from the sidebar");
+    cy.get("@sidebarOnboardingLink").should("not.exist");
+
+    cy.log("Navigation is redirected to the home page");
+    cy.get("@accordionElement").should("not.exist");
```

Comment:
I'm not sure I understand - what would be the scenario in which a component re-renders? Unless I misunderstood you, we should be pretty safe because of `cy.location("pathname").should("eq", "/");` in the next line.

If it makes it more straight-forward, I can swap the order of these two assertions. The positive one should almost always be the first one, anyway.